ToolkitApp/tabs/src/components/sample/MainContent.jsx

```diff
@@ -13,17 +13,23 @@ const MainContent = () => {
     const [thirdSectionActive, toggleThirdSection] = useState(true);
     const [fourthSectionActive, toggleFourthSection] = useState(true);
 
+    const [connectedResourcesList, setResourceList] = useState([]);
+    const [title, setTitle] = useState(" ");
+
+
```

Comment:
Is there a way to put this inside a component, abstract it away?

Reply:
0cb3efb2c1bdb37baf52d2c9b913480b68c0a205